### ISSUE TITLE
remix

### ISSUE THREAD

**Issue by gedw99** [Darkness4/fc2-live-dl-go#13]
this is a drop in replacement for the remix code.

https://github.com/asticode/go-astiav

its one to one c to golang api like your code.

for example:
https://github.com/search?q=repo%3Aasticode%2Fgo-astiav%20C.AV_ERROR_MAX_STRING_SIZE&type=code

**Comment by Darkness4** [comment]
I won't be using a third-party library for remux for several reasons:

1. `go-astiav` is only compatible with ffmpeg 5.1.2, which adds coupling to a specific version of ffmpeg.
2. I don't need the full ffmpeg binding to simply concatenate packets into an mp4. 
3. The current configuration allows the cross-compilation of static binaries under linux and windows, and allows the use of non-gpl ffmpeg libraries.

Basically, the project is already loosely coupled to ffmpeg, allowing static cross-compilation from an older version of ffmpeg, up to the most recent one (6.0) which is vital when building for multiple operating systems.

**Comment by gedw99** [comment]
Totally agree . YAGNI principles.
